src/cookbook/submit-transactions.md

```diff
@@ -11,28 +11,23 @@ head:
 
 # How to Submit a Transaction
 
-```rust
-//Precondition: the Client variable must be initialized
-    
-//First we need to build an instruction expression that we want to submit
-//The instruction expression structures have a self-explanatory name
-//and you can easily understand what each of them does.
-//As an example:
-//* BurnExp - burns asset quantity
-//* MintExpr - mints asset quantity
-//* RegisterExpr - registers any Registrable object like
-//** Account, Asset, Asset definition, Trigger etc.
-//As an example we will take the MintExpr
-
-let asset_definition_id = AssetDefinitionId::from_str("coolAsset#wonderland").unwrap();
-let asset_id: AssetId = AssetId::new(asset_definition_id, AccountId::from_str("alice@wonderland").unwrap());
-let asset_quantity: Fixed = Fixed::from_str("7").unwrap();
+See [Create Transactions](create-transactions.md) to learn how to create
+a `SignedTransaction` used in the examples below.
 
-let some_expression = MintExpr::new(asset_quantity, asset_id);
+```rust
+fn submit_transaction_do_not_wait_for_approval(
+    iroha: &Client,
+    transaction: &SignedTransaction
+) {
+    // If the transaction is rejected, the method panics
```

Comment:
Is this correct? AFAIK it will panic if transaction is _invalid_ and Iroha doesn't even accept it. And whether it is _approved_ or _rejected_ is decided later.

Reply:
Yep, I overlooked that, thanks.